react/Table/Virtuoso/Cell.jsx

```diff
@@ -0,0 +1,41 @@
+import cx from 'classnames'
+import React from 'react'
+
+import TableCell from '../../TableCell'
+import { makeStyles } from '../../styles'
+
+const useStyles = makeStyles({
+  root: {
+    width: ({ column }) => column.width,
+    maxWidth: ({ column }) => column.maxWidth
+  }
+})
+
+const Cell = ({ row, columns, column, children }) => {
+  const classes = useStyles({ column })
+
+  return (
+    <TableCell
+      key={column.id}
+      classes={classes}
+      className={cx({ sortable: column.sortable !== false })}
```

Comment:
Why not `{ sortable: column.sortable }` ?

Reply:
you mean `className={cx({ sortable: column.sortable })}` ? Because this way it doesn't deal with `undefined` and we have to be specific in the declaration, the `sortable` attr should be optional in my opinion.

And why not simply `{ sortable: column.sortable }` because className has to be a string.